packages/extension/src/companion/CompanionPermission.tsx

```diff
@@ -23,14 +18,11 @@ const CompanionPermissionComponent = (
   const { requestContentScripts } = useExtensionPermission({
     origin: 'companion permission popup',
   });
-  const { flags } = useContext(FeaturesContext);
-  const link = getFeatureValue(Features.CompanionPermissionLink, flags);
-  const button = getFeatureValue(Features.CompanionPermissionButton, flags);
-  const title = getFeatureValue(Features.CompanionPermissionTitle, flags);
-  const description = getFeatureValue(
-    Features.CompanionPermissionDescription,
-    flags,
-  );
+  const link = 'Overview Video';
+  const button = 'Activate companion';
+  const title = 'Try the new companion feature!';
+  const description =
+    "We'll ask for extra permissions so we can show the companion directly on the original content!";
```

Comment:
Should we now take this to the label file that Tom has created?

Reply:
Ah good point 👍

Reply:
Although cannot use it yet, because it's merged to MVP instead of main.
So let's rather retro fix later down the line.

Reply:
Agreed. Let's do that for the moment 🚀